tests/ignore_hydrogens.rs

```diff
@@ -0,0 +1,31 @@
+// Test reading PDB file and discarding all hydrogen atoms.
+
+use pdbtbx::*;
+
+#[test]
+fn main() {
```

Comment:
can you give the test a more descriptive name?

Reply:
Done!